Dockerfile

```diff
@@ -148,8 +148,7 @@ RUN luarocks install luacheck
 # NPM setup
 # Extract dependencies from coala-bear package.json
 # typescript is a peer dependency
-RUN npm install -g typescript \
-    $(sed -ne '/~/{s/^[^"]*"//;s/".*"~/@/;s/",*//;p}' coala-bears/package.json)
```

Comment:
the sed expression must have had a purpose, we still use the same JSON - why remove it? This deserves a mention in the commit message